application-admintools-default/src/main/java/com/xwiki/admintools/internal/data/identifiers/TomcatIdentifier.java

```diff
@@ -82,6 +85,15 @@ public void updatePossiblePaths()
             String.format("%s/webapps/xwiki/WEB-INF/", this.serverPath) };
     }
 
+    @Override
+    public String getServerNameAndVersion()
```

Comment:
Maybe better to have a `getServerMetadata` that return a map with these 2 values (for now), because you might need in the future to return other things. For now you'll concatenate these 2 in velocity
I think in general it's best that when you have multiple values, to return a list /map with them and do the concatenation where it's needed